### PR TITLE
Add org.freedesktop.Sdk.Extension.node20

### PR DESCRIPTION
**Pull request by razzeee**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

First try to build this froze my laptop - but it's basically a copy job of the node18 package.

### CODE REVIEW

**Comment by bbhtt** [comment]
bot, build org.freedesktop.Sdk.Extension.node20

**Comment by bbhtt** [comment]
Can you also symlink node-gyp to `/app/bin`? It's in `lib/node_modules/npm/bin/node-gyp-bin/node-gyp`

There are some leftover man pages too.

File: org.freedesktop.Sdk.Extension.node20.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      - --with-intl=system-icu
+
+  - name: yarn
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/{bin,share}
+      - cp -a . ${FLATPAK_DEST}/share/yarn
+      - ln -sr ${FLATPAK_DEST}/share/yarn/bin/yarn ${FLATPAK_DEST}/bin/yarn
+      - ln -sr ${FLATPAK_DEST}/share/yarn/bin/yarnpkg ${FLATPAK_DEST}/bin/yarnpkg
+      - ln -sr lib/node_modules/npm/bin/node-gyp-bin/node-gyp ${FLATPAK_DEST}/bin/node-gyp
```
Comment (bbhtt) [inline org.freedesktop.Sdk.Extension.node20.yaml:48]:
```suggestion
      - ln -sr ${FLATPAK_DEST}/lib/node_modules/npm/bin/node-gyp-bin/node-gyp ${FLATPAK_DEST}/bin/node-gyp
```

**Comment by razzeee** [comment]
> There are some leftover man pages too.

How did you check that?

**Comment by hfiguiere** [comment]
look at the content of the build dir. everything in `files`

**Comment by razzeee** [comment]
Guess I will have to buy more RAM to figure this out then :D

**Comment by bbhtt** [comment]
You can also install the test build and check `~/.local/share/flatpak/runtime/<id>/23.08/files`

**Comment by razzeee** [comment]
bot, build org.freedesktop.Sdk.Extension.node20

**Comment by razzeee** [comment]
Man pages and docs should be gone now too

**Pull request closed without merging**